qutebrowser/browser/qutescheme.py

```diff
@@ -30,9 +30,11 @@
 import textwrap
 import mimetypes
 import urllib
+import collections
 
 import pkg_resources
 from PyQt5.QtCore import QUrlQuery, QUrl
+import sip
```

Comment:
I hope I'm not being to nitpicky, but I'd prefer this to be before `from ...` imports.

Reply:
No problem